org.gabmus.unifydmin.json

```diff
@@ -0,0 +1,410 @@
+{
+    "app-id": "org.gabmus.unifydmin",
+    "command": "unifydmin",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--filesystem=home:ro",
```

Comment:
Why does the app requires those permissions? Same for SSH.

Reply:
- home access: it was a leftover from another project, sorry
- `~/.ssh` access, this application is an administration utility similar to cockpit in a way, that allows to control different servers all in one place. It uses ssh to connect to those servers using a library called fabric. fabric automatically logs in using ssh keys if they are available, thus the read access to `~/.ssh`. Removing it (I tried) prevents fabric from connecting, making the whole application practically useless.
- talking to org.freedesktop.Flatpak: allows to run non sandboxed commands to open a terminal with an ssh connection in your favorite terminal (user configurable). The default is xterm.